waku/waku_metadata/protocol.nim

```diff
@@ -13,77 +13,82 @@ import
   libp2p/stream/connection,
   libp2p/crypto/crypto,
   eth/p2p/discoveryv5/enr
-import
-  ../common/nimchronos,
-  ../common/enr,
-  ../waku_core,
-  ../waku_enr,
-  ./rpc
+import ../common/nimchronos, ../common/enr, ../waku_core, ../waku_enr, ./rpc
```

Comment:
Not sure I like this style. Seams harder to read than vertical

Reply:
Yes, I agree that is easier to read in vertical style. Also better for comparing. We can submit a request to the `nph` repo.

Reply:
I kinda think the same. Vertical and sorted by name/directory would be great and/or grouped.